packages/web-api/src/WebClient.ts

```diff
@@ -344,10 +344,11 @@ export class WebClient extends Methods {
             await delay(retrySec * 1000);
             // resume the request queue and throw a non-abort error to signal a retry
             this.requestQueue.start();
-            throw Error('A rate limit was exceeded.');
+            // TODO: We may want to have more detailed info such as team_id, params except tokens, and so on.
+            throw Error(`A rate limit was exceeded (url: ${url}, retry-after: ${retrySec})`);
           } else {
             // TODO: turn this into some CodedError
-            throw new AbortError(new Error('Retry header did not contain a valid timeout.'));
+            throw new AbortError(new Error(`Retry header did not contain a valid timeout (url: ${url})`));
```

Comment:
@srajiang @stevengill 
Sorry for changes after the reviews but I just updated this one as well. It should be fine too.

Reply:
Yes, this looks fine!